src/Compilers/Core/MSBuildTaskTests/MSBuildTaskTests.csproj

```diff
@@ -15,7 +15,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <SolutionDir Condition="'$(SolutionDir)' == '' OR '$(SolutionDir)' == '*Undefined*'">..\..\..\..\</SolutionDir>
     <RestorePackages>true</RestorePackages>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Yes for future (in fact, I think @agocke already did it there).  We can't for master for shipping projects, but since this is non-shipping, I felt comfortable updating it.